examples/algorithms/kirchhoffs_circuit_laws.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
oops -- I see that this file is in the examples directory already...

Should we remove the ```if __name__ == "__main__":```?